src/redux/features/observations/observationsSelectors.ts

```diff
@@ -30,7 +30,7 @@ export const selectPlantingSiteObservationsResults = createCachedSelector(
       const matchesSite = plantingSiteId === -1 || observationResults.plantingSiteId === plantingSiteId;
       const matchesState =
         (!status?.length && observationResults.state === 'Completed') ||
-        status?.indexOf(observationResults.state) !== -1;
+        (status && status.indexOf(observationResults.state) !== -1);
```

Comment:
The "fix" here is the parentheses, right? Removing the '?' and doing a separate validation for status should not change anything I guess

Reply:
so if status is undefined, the check previously returns true (since undefined is !== -1)
that was wrong, we want to check for the indexOf result to be a number (not undefined), hence the `status && status.indexOf` fix